packages/fiori/src/Timeline.js

```diff
@@ -16,6 +18,28 @@ const metadata = {
 	tag: "ui5-timeline",
 	languageAware: true,
 	managedSlots: true,
+	properties: /** @lends  sap.ui.webcomponents.fiori.Timeline.prototype */ {
+		/**
+		 * Select orientationn.
+		 *
+		 * <br><br>
+		 * <b>Note:</b>
+		 * Available options are:
+		 * <ul>
+	 	* <li><code>Vertical</code></li>
+	 	* <li><code>Horizontal</code></li>
+		 * </ul>
+		 *
+		 * @type {TimelineLayout}
+		 * @defaultvalue "Vertical"
+		 * @since 1.0.0-rc.15
+		 * @public
+		 */
+		axisOrientation: {
```

Comment:
Isn't "layout" simpler or you did have an API discussion and agreed on that name.

Reply:
Done